### PR TITLE
Add static route back to internal-to-storage router on storage nodes

### PR DESCRIPTION
**Pull request by jovial**

### CODE REVIEW

File: etc/kayobe/inventory/group_vars/ceph/network-interfaces [hunk 1]
```diff
@@ -36,6 +36,11 @@ internal_net_interface: "{{ br_net_interface }}.{{ internal_net_vlan }}"
 # Storage network IP information.
 storage_net_interface: "{{ br_net_interface }}.{{ storage_net_vlan }}"
 
+# Add a route back to the internal-to-storage router
+storage_net_routes:
+  - cidr: "10.215.215.0/24"
```
Comment (JohnGarbutt) [inline etc/kayobe/inventory/group_vars/ceph/network-interfaces:41]: Looks good, should we add the 10.218 in there too?

**Review by JohnGarbutt — APPROVED** [review]

**Review by JohnGarbutt — CHANGES_REQUESTED** [review]
Actually given the recent changes we removed the 10.215 network, so we need to add the route for the 10.208 networking, I think.

**Comment by paulbrowne** [comment]
I added the static route from storage 206 to provider network 218 in this PR to capture it in the config;

https://github.com/RSE-Cambridge/cumulus-kayobe-config/pull/79/files#r332463010

**Comment by jovial** [comment]
Thanks, looks like I can close this one now then.

**Pull request closed without merging**